skimage/color/colorconv.py

```diff
@@ -933,7 +933,7 @@ def xyz2lab(xyz, illuminant="D65", observer="2"):
     a = 500.0 * (x - y)
     b = 200.0 * (y - z)
 
-    return np.concatenate([x[..., np.newaxis] for x in [L, a, b]], axis=-1)
+    return np.dstack([L, a, b])
```

Comment:
https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.dstack.html

> This function continues to be supported for backward compatibility, but you should prefer np.concatenate or np.stack. The np.stack function was added in NumPy 1.10.

Reply:
Also, this change will not work for 4-D arrays (3-D + color channels), which are supported with the current version. Sorry...

Reply:
However, 
```
np.stack([L, a, b], axis=-1)
```
would work and be an improvement in terms of clarity.

Reply:
```suggestion
    return np.stack([L, a, b], axis=-1)
```